persistence/src/commonMain/db_user/com/wire/kalium/persistence/MessageDrafts.sq

```diff
@@ -0,0 +1,29 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import com.wire.kalium.persistence.dao.message.MessageEntity.Mention;
+import kotlin.collections.List;
+
+CREATE TABLE MessageDrafts (
+      conversation_id TEXT AS QualifiedIDEntity NOT NULL,
+      text TEXT,
+      edit_message_id TEXT,
```

Comment:
Should the reply and edit also be also a FOREIGN KEY to a message ID ? in that case if the message got deleted or changed we can change how draft will react 
we can for example set the edit_message_id or qouted_message_id to null when deleted on changed 

https://www.sqlite.org/foreignkeys.html

Reply:
Good idea